subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/corefeature/LargeDependencyGraphPerformanceTest.groovy

```diff
@@ -94,26 +86,46 @@ class LargeDependencyGraphPerformanceTest extends AbstractCrossVersionPerformanc
         stopServer()
     }
 
-    private static BuildExperimentListener createArgsTweaker(String baseline) {
-        new BuildExperimentListener() {
-            @Override
-            void beforeExperiment(BuildExperimentSpec experimentSpec, File projectDir) {
-                GradleInvocationSpec invocation = experimentSpec.invocation as GradleInvocationSpec
-                if (invocation.gradleDistribution.version.version != baseline) {
-                    invocation.args << '-Dorg.gradle.advancedpomsupport=true'
-                }
-            }
+    def "resolve large dependency graph with advanced pom support"() {
```

Comment:
Now that we have several similar examples, we should consider using `where:`  and `@Unroll` to define the different test scenarios.

Reply:
Going to use this as an excuse to learn that feature

Reply:
Check out this example: https://github.com/gradle/gradle/blob/5a1a8781d8d0fd1c866d64d1e7bacffd4294da55/subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/corefeature/RichConsolePerformanceTest.groovy#L33-L55